libraries/chain/event_evaluator.cpp

```diff
@@ -62,7 +95,17 @@ object_id_type event_create_evaluator::do_apply(const event_create_operation& op
          event_obj.season = op.season;
          event_obj.start_time = op.start_time;
          event_obj.event_group_id = event_group_id;
+         //if( op.extensions.value.new_manager ) { // NORMALLY SET MANAGER HERE BUT DOESNT WORK
```

Comment:
Interestingly, if using the code here, the manager is **not** set in the object properly.
This needs investigation
@pmconrad: Do you maybe know what might cause this?

Reply:
Nice one. :-)
`event_obj` implements its own assignment operator and copy constructor. Both are missing the new manager field.
https://github.com/blockchainprojects/peerplays/blob/feature/manager-for-sports-and-underlying/libraries/chain/event_object.cpp#L376-L383
https://github.com/blockchainprojects/peerplays/blob/feature/manager-for-sports-and-underlying/libraries/chain/event_object.cpp#L392-L401

Reply:
Ahh okay I see. Will change that. Thanks